tools/installers/install_chime7task1.sh

```diff
@@ -0,0 +1,10 @@
+./install_s3prl.sh
```

Comment:
installation requirements for the challenge baseline are in optional installation package

Reply:
Will do